community/kernel/src/main/java/org/neo4j/storageengine/api/StorageStatement.java

```diff
@@ -68,19 +69,21 @@
      * to place the cursor over the first item and then more calls to move the cursor through the selection.
      *
      * @param nodeId id of node to get cursor for.
+     * @param assertOpen to check if source transaction is still open on each property value fetch
```

Comment:
why do we need to say that assertOpen checking source transaction? shouldn't comment have a more generic expectation from what assert open doing? (this comment related to all new assertOpen params introduced)